### PR TITLE
remove calculate dependency on mapreduce, ndb

### PR DESCRIPTION
**Pull request by colincadams**

### CODE REVIEW

File: recidiviz/calculator/recidivism/identifier.py [hunk 1]
```diff
@@ -62,6 +59,9 @@ def find_recidivism(person, include_conditional_violations=False):
 
     Args:
         person: a person to determine recidivism for.
```
Comment (jessex) [inline recidiviz/calculator/recidivism/identifier.py]: This line can be removed now.

File: recidiviz/calculator/recidivism/pipeline.py [hunk 2]
```diff
@@ -23,21 +23,24 @@ class that initiates pipelines on web requests.
...
+    'total_people_mapped': 0,
+    'total_metric_combinations_mapped': 0,
+    'unique_metric_keys_reduced': 0,
+    'total_records_reduced': 0,
+    'total_recidivisms_reduced': 0,
+}
+
 
-def map_person(person):
+# placholder - should be mapreduce execution id
```
Comment (jessex) [inline recidiviz/calculator/recidivism/pipeline.py]: `placeholder`

**Comment by jessex** [comment]
I think there's more configuration we can remove now, including but not limited to:
* A ton of library code under `/lib`
* The /_ah/* config from base.yaml
* The module and version config from local_app.yaml
* The mapreduce.yaml file

Take a look at this [commit](https://github.com/Recidiviz/recidiviz-data/commit/641a9e792e69eaaf889e460892e31e2fdfcce734) from when calculation was first added to the repository for context.

**Comment by colincadams** [comment]
alright I removed what I believe is everything except for the `libs/` changes. It's hard to tell if any other parts of the system have started to depend on those packages since they were added or not and we need to just delete the whole `libs/` directory eventually.

**Comment by colincadams** [comment]
Just kidding, I removed the packages from `libs/` as well. Looks like the only one that was still depended on is `pbr`.

**Pull request merged**